scripts/build.ps1

```diff
@@ -66,7 +66,10 @@ if (!$?) {
     exit 1
 }
 
-& ninja -v -j 16 -C $buildoutput v8jsi jsitests | Tee-Object -FilePath "$SourcesPath\build.log"
+# We'll use 2x the number of cores for parallel execution

```

Comment:
>2x [](start = 12, length = 2)

why? we shouldn't be meddling with such low level system stuff. can't system figure out most optimal parallelism for us?

Reply:
With a hardcoded 16 parallelism, some build agents were choking (effectively grinding to a halt). This tries to reduce the number of parallel builds on low-end devices like that.